ibm/mas_devops/common_tasks/backup_restore/copy_storage_files_to_pod.yml

```diff
@@ -21,13 +21,12 @@
         - item.dest_folder is defined and item.dest_folder | length > 0
       shell: >-
         oc exec {{ masbr_cf_pod_name }} -c {{ masbr_cf_container_name }} -n {{ masbr_cf_namespace }} -- bash -c
-        'temp_dest_folder={{ [item.dest_folder, masbr_job_version] | path_join }} &&
-        mkdir -p ${temp_dest_folder} &&
-        oc cp --retries=50 -c {{ masbr_cf_container_name }}
-        {{ [masbr_storage_job_folder, item.src_folder] | path_join }}
-        {{ masbr_cf_namespace }}/{{ masbr_cf_pod_name }}:${temp_dest_folder} &&
-        mv -f ${temp_dest_folder}/* {{ item.dest_folder }} &&
-        rm -rf ${temp_dest_folder}'
+          'temp_dest_folder={{ [item.dest_folder, masbr_job_version] | path_join }} && mkdir -p ${temp_dest_folder}'
+        && oc cp --retries=50 -c {{ masbr_cf_container_name }}
+          {{ [masbr_storage_job_folder, item.src_folder] | path_join }}
+          {{ masbr_cf_namespace }}/{{ masbr_cf_pod_name }}:${temp_dest_folder}
+        && mv -f ${temp_dest_folder}/* {{ item.dest_folder }}
+        && rm -rf ${temp_dest_folder}
```

Comment:
I read this whole command as: exec into the masbr_cf_pod_name pod and create a temp folder, return from container. Then on the host running this task run oc cp to copy a folder to the masbr_cf_pod_name and that create temp folder. That is fine so far, but the problem now is that the mv from the temp folder to another is not done on the masbr_cf_pod_name pod but on the host running this task which doesn't have the temp folder. The rm is also done on the wrong host.